src/linear_algebra/free_module.lean

```diff
@@ -155,3 +155,49 @@ of_basis (basis.of_vector_space R M)
 end division_ring
 
 end module.free
+
+/-! ### Standard inclusions
+We define in this section the standard inclusion `fin n → R →ₗ[R] (fin (n + m) → R)`.
+-/
+
+variables (ι : Type v) (η : Type w)
+
+/-- For any type `R` with `0`, we extend any `f : ι → R` to `(ι ⊕ η) → R`. -/
+def ext_zero [has_zero R] : (ι → R) → ((ι ⊕ η) → R) := λ f i, sum.cases_on i (λ j, f j) (λ j, 0)
```

Comment:
This has nothing to do with `free_module`, or linear algebra, so can't go in this file.